### ISSUE TITLE
Code formatter, anon functions

### ISSUE THREAD

**Issue by infu** [dfinity/vscode-motoko#57]
Code formatter messes up anonymous functions as function parameters, puts them in one line and when there are comments, breaks the code.

**Comment by infu** [comment]
```
job_add( func () : async () {
                
                    await job_canister_start({slot = new_history_slot});
                    //await job_oracle_set({slot = new_history_slot; oracle = _oracle});
                    await job_conf_set({slot = new_history_slot; config = _conf});
});
```
will result in 
```
job_add( func () : async () {await job_canister_start({slot = new_history_slot}); //await job_oracle_set({slot = new_history_slot; oracle = _oracle});  await job_conf_set({slot = new_history_slot; config = _conf}); });
```

**Comment by rvanasa** [comment]
This will be fixed in the next release (`v0.4.7`). Cheers!

 In the meantime, you can use a `// prettier-ignore` comment to work around this.
